CIP-_/README.md

```diff
@@ -0,0 +1,209 @@
+---
+CIP: "?"
+Title: Optimized Coin Selection Algorithms for Cardano
```

Comment:
```suggestion
Title: Coin Selection - Optimized Random Improve
```
This first part of this title is far more general than you're presenting here, and the phrase "for Cardano" is unncecessary.  I'll suggest this title & we'll also see how the same title looks in the PR title (if changed further, they should be consistent... and both as few words as possible to convey your idea completely & succinctly).

I'm moving the Coin Selection term first because we have other groupings of CIPs and for which it is proving help to have the general term first & then the specific term (e.g. the Web-Wallet Bridge CIPs) and we're likely to have more multiple coin selection algorithms proposed as well.

Reply:
@NicholasMaselli committing since no objections & we should have a standardised title for review at CIP meeting tomorrow.